bin/run-pacta

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+cp -r ../input/*.yml ./working_dir/10_Parameter_File
+cp -r ../input/*.csv ./working_dir/20_Raw_Inputs
+
+Rscript --vanilla -e "source('R/run_pacta.R'); run_pacta()"
+cp -r ./working_dir ../output
```

Comment:
TODO: ../output has new files with restrictive permissions. User's can't delete them unless they use sudo. CJ used umask 000 to relax such permissions, but that didn't seem to work for me. I need to try again.

Reply:
For the record Alex suggested `chown` instead of `umask`.

Reply:
Follow at https://github.com/2DegreesInvesting/pactaCore/issues/22